app/models/policy_area.rb

```diff
@@ -6,4 +6,6 @@ class PolicyArea < ActiveRecord::Base
   validates :name, presence: true, uniqueness: true
   validates :slug, presence: true, uniqueness: true
   validates :description, presence: true
+
+  serialize :organisation_content_ids, Array
```

Comment:
Oooh, yes we can - ActiveRecord even has built-in support for Postgres array type! http://edgeguides.rubyonrails.org/active_record_postgresql.html#array